app/Helpers/general.php

```diff
@@ -375,4 +378,29 @@ function parseUrls($content)
         return array_map($checkAbsoluteUrl,  $matches[1]);
     }
     return [];
+}
+
+/**
+ * Returns an estimated reading time in a string
+ * idea from @link http://briancray.com/posts/estimated-reading-time-web-design/
+ * @param  string $content the content to be read
+ * @param int $wpm
+ * @return string          estimated read time eg. 1 minute, 30 seconds
+ **/
+function getEstimateReadingTime($content, $locale, $wpm = 200) {
```

Comment:
Is this a typo? (if not, feel free to ignore)
```suggestion
function getEstimatedReadingTime($content, $locale, $wpm = 200) {
```

Reply:
BTW, do we need `$locale` and `$wpm` here?

`$locale` does not seem to be used & `$wpm` could probably be a constant.

Reply:
I'd say that @martin-havala meant to pass `$locale` to `trans_choice`

Reply:
And would that be even necessary? Doesn't `trans_choice()` use the current app locale by default?

Reply:
It can be optional (`=== null`). Just to keep it universal if we want to use it somewhere else e.g. when sending e-mails or where the locale is not set globally since it's a general helper.